patch-package
--- a/node_modules/react-cosmos-voyager2/dist/server/extract-components-from-fixture-file.js
+++ b/node_modules/react-cosmos-voyager2/dist/server/extract-components-from-fixture-file.js
@@ -45,7 +45,7 @@ var extractComponentsFromFixtureFile = exports.extractComponentsFromFixtureFile
             ast = babylon.parse(code, {
               sourceType: 'module',
               // XXX: Does using all plugins have any disadvantage?
-              plugins: ['jsx', 'flow', 'typescript', 'objectRestSpread', 'classProperties', 'asyncGenerators', 'dynamicImport']
+              plugins: ['jsx', 'typescript', 'objectRestSpread', 'classProperties', 'asyncGenerators', 'dynamicImport']
             });
             _context.next = 13;
             break;
@@ -212,7 +212,7 @@ var _util = require('util.promisify');
 
 var _util2 = _interopRequireDefault(_util);
 
-var _babylon = require('babylon');
+var _babylon = require('@babel/parser');
 
 var babylon = _interopRequireWildcard(_babylon);
 
